src/main/java/io/stargate/sgv2/jsonapi/api/model/command/clause/filter/ComparisonExpression.java

```diff
@@ -33,6 +33,32 @@ public List<FilterOperation<?>> getFilterOperations() {
     return filterOperations;
   }
 
+  /** This method will be called when not operation is pushed down */
+  public void flip() {
+    List<FilterOperation<?>> filterOperations = new ArrayList<>(this.filterOperations.size());
+    for (FilterOperation<?> filterOperation : this.filterOperations) {
+      final FilterOperator flipedOperator = filterOperation.operator().flip();
+      JsonLiteral<?> operand =
+          getFlippedOperandValue(filterOperation.operator(), filterOperation.operand());
+      filterOperations.add(new ValueComparisonOperation<>(flipedOperator, operand));
+    }
+    this.filterOperations = filterOperations;
+  }
+
+  /**
+   * This method is used to flip the operand value when not operator is applied, e.g. $exists, $size
+   */
+  private JsonLiteral<?> getFlippedOperandValue(FilterOperator operator, JsonLiteral<?> operand) {
+    if (operator == ElementComparisonOperator.EXISTS) {
+      return new JsonLiteral<Boolean>(!((Boolean) operand.value()), operand.type());
+    } else if (operator == ArrayComparisonOperator.SIZE) {
+      return new JsonLiteral<BigDecimal>(
+          ((BigDecimal) operand.value()).multiply(new BigDecimal(-1)), operand.type());
```

Comment:
If this is done to flip between 1 and -1, more efficient way would be to use `BigDecimal.negate()` (https://docs.oracle.com/javase/8/docs/api/java/math/BigDecimal.html#negate-- )